packages/flutter/lib/src/painting/flutter_logo.dart

```diff
@@ -12,6 +12,7 @@ import 'package:flutter/foundation.dart';
 import 'basic_types.dart';
 import 'box_fit.dart';
 import 'decoration.dart';
+import 'edge_insets.dart';
```

Comment:
?

Reply:
Maybe something (decoration.dart?) exports EdgeInsets but not EdgeInsetsDirectional?  The analyzer was made without this import.

Reply:
But you didn't make any other changes to this file. I don't understand.

Reply:
Ah, sorry.  I tried to RTL-ize FlutterLogoDecoration and backed out.  I'll remove this diff.